bakerydemo/templates/locations/locations_index_page.html

```diff
@@ -1,18 +1,36 @@
 {% extends "base.html" %}
-{% load wagtailcore_tags %}
-{% load wagtailimages_tags %}
+{% load wagtailcore_tags navigation_tags wagtailimages_tags %}
 
 {% block content-header %}
-    {% include "base/include/header.html" %}
```

Comment:
@heymonkeyriot I had gone through and moved all of the headers into a common header include. It looks like that has been lost here. Intentional? If not, can we preserve or refactor that system? (OK if we do that after this lands).